### ISSUE TITLE
1-actualizar-datatable-eliminar

### ISSUE THREAD

**Issue by gasparoberti** [gasparoberti/curso-java#1]
corregir la línea location.reload() del archivo usuarios.js para que al eliminar solo recargue el datatable en vez de toda la página.

buscar algo parecido a esto: (que no funciona)

$('#usuariosTable').DataTable().ajax.reload();

**Comment by gasparoberti** [comment]
modificar interfaz para que use el modal de bootstrap en vez del confirm de js



<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#staticBackdrop">
  Launch static backdrop modal
</button>


<div class="modal fade" id="staticBackdrop" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="staticBackdropLabel">Modal title</h5>
        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
      </div>
      <div class="modal-body">
        ...
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Understood</button>
      </div>
    </div>
  </div>
</div>
